libs/feature/map/src/lib/map-context/map-context.model.ts

```diff
@@ -23,3 +24,4 @@ export interface MapContextViewModel {
   center: [number, number]
   zoom: number
 }
+export type MapContextExtentModel = [number, number, number, number]
```

Comment:
I think we should document that this is expressed in the map view projection, so web mercator for now

Reply:
yes I agree, where should I do so, here in a comment or in a readme?

Reply:
a comment is enough for now I think

Reply:
ok, I added that